modular_skyrat/modules/modular_items/code/game/objects/items/ciggies.dm

```diff
@@ -0,0 +1,14 @@
+/obj/item/storage/fancy/cigarettes/cigars/havana/lyricalpaws
+	name = "Bright Cosmos cigar case"
+	desc = "A case of imported Bright Cosmos cigars, very hard to find usually. Renowned for their quick dispersal smoke, strong taste, and \"bigger on the inside\" cases."
+	spawn_type = /obj/item/clothing/mask/cigarette/cigar/havana/lyricalpaws
+
+/obj/item/clothing/mask/cigarette/cigar/havana/lyricalpaws
+	name = "Bright Cosmos cigar"
+	desc = "Per aspera ad astra."
+	icon_state = "cigar2off"
+	icon_on = "cigar2on"
+	icon_off = "cigar2off"
+	list_reagents =list(/datum/reagent/drug/nicotine = 15, /datum/reagent/consumable/menthol = 7)
+	pollution_type = /datum/pollutant/bright_cosmos
+	lung_harm = 0
```

Comment:
A buffed havana cigar would fall under balance changes.

Havana cigars are a donator item currently as well, this should be identical aside from flavor additions.

Reply:
Yeah, remove this.

Reply:
Havana aren't donator exclusive, and the only issue I see is the lung harm, which is about as mechanical as other donator items (Look at the hexnut ring)

Reply:
Just because other people snuck in gameplay changes doesn't mean you get to as well. Remove it.

Reply:
It wasn't "snuck in" it was added with intent

Reply:
Remove the lung damage change or I'm removing this for you in a follow up. I'll address the other shitty donator item myself.

Reply:
if useroth okay'd it, its fine

Reply:
and, its barely a gameplay change, its a ckey locked donator item, unless I'm mistaken, nicotine and menthol not only have no positive effect, they have *negative* ones

this makes them purely cosmetic

but, my view doesn't matter, it'd be up to @Useroth about donator items, "shitty" or not

Reply:
Menthol only stops coughing, nicotine has a very minor stun reduction and induces nicotine addiction, I'm entirely willing to remove the nicotine and moodlet for smoking if it means I can get it through

Reply:
you do not get to have free stun resist mouthsticks with zero penalty for usage

Reply:
if useroth okay'd it

Reply:
I'm already speaking to useroth about it, and will implement the changes **they** want.

Reply:
good!